_includes/week.md

```diff
@@ -28,7 +28,8 @@
 
 **Resources**:
 - <i class="fas fa-clipboard"></i> [Notes]({{ c.notes }})
-- <i class="fab fa-youtube"></i> {% if c.recording %} [Recording]({{ c.recording }}) {% else %} Recording available on the [OLS YouTube channel]({{ site.youtube }}) after the call {% endif %}
+- <i class="fab fa-youtube"></i> {% if c.recording %} Resources
+<iframe src="{{ c.recording }}"></iframe> {% else %} Recording available on the [OLS YouTube channel]({{ site.youtube }}) after the call {% endif %}
```

Comment:
It would be nice to make the video bigger in the page there

Reply:
Maybe an idea: having on the side of the call detail

Reply:
💭 That's a  good suggestion. I was concerned that having a bigger video might give us responsiveness issues.

I'll try to do a sample soon and get your thoughts on it. Thanks!